src/workerd/jsg/ser.c++

```diff
@@ -133,4 +133,23 @@ v8::Local<v8::Value> structuredClone(
   return des.readValue();
 }
 
+kj::Array<kj::byte> serializeV8Rpc(jsg::Lock& js, v8::Local<v8::Value> value) {
```

Comment:
I'd rather not have methods specific to "Rpc" in the jsg definitions here. There doesn't appear to be anything rpc specific in these? And since they're only used in the js-rpc impl, I'd rather these utilities go there.

Reply:
Yeah I agree. It looks like each of these helpers is called in exactly one place in the code. Seems like those places should just use `jsg::Serializer` directly, or if anything these helper functions could be local to that file.

Reply:
"Rpc" was added because I was fighting the linker, thought maybe some name conflict but it was a header issue. Fully intended on removing "Rpc". My hope for these funcs was to use them on both client and server (we still aren't (de)serializing args) + replace existing methods used for Hibernatable WebSocket serialization since it's practically the same.

Can circle back to this, I'll add a `TODO(now)` so I don't forgot.

Reply:
https://github.com/cloudflare/workerd/compare/1e60d2d5059fd230db0c16802aa7385e1f49a3ff..1697bf72c23c1b9a3066534a6956496da24ee044

Moved it to a private namespace.